_topics/en/statechains.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Statechains
+
+## Optional.  Shorter name to use for reference style links e.g., "foo"
+## will allow using the link [topic foo][].  Not case sensitive
+# shortname: foo
+
+## Optional.  An entry will be added to the topics index for each alias
+#aliases:
+#  - Foo
+
+## Required.  At least one category to which this topic belongs.  See
+## schema for options
+categories:
+  - Contract Protocols
+
+## Optional.  Produces a Markdown link with either "[title][]" or
+## "[title](link)"
+primary_sources:
+    - title: Statechains
+      link: https://medium.com/@RubenSomsen/statechains-non-custodial-off-chain-bitcoin-transfer-1ae4845a4a39
+
+    - title: Statechains with ECDSA and without eltoo
+      link: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2020-March/017714.html
+
+## Optional.  Each entry requires "title", "url", and "date".  May also use "feature:
+## true" to bold entry
+optech_mentions:
+  - title: "Bitcoin Core contributor meeting discussion topic: blind statechains"
+    url: /en/newsletters/2019/06/12/#assumeutxo
+
+  - title: Discussion about implementing statechains without schnorr or eltoo
+    url: /en/newsletters/2020/04/01/#implementing-statechains-without-schnorr-or-eltoo
+
+  - title: Transcript abuot watchtowers and their usefulness for statechains
+    url: /en/newsletters/2020/07/01/#watchtowers-and-bolt13
+
+  - title: Transcript of discussion about multiple topics, including statechains
+    url: /en/newsletters/2020/09/02/#sydney-meetup-discussion
+
+## Optional.  Same format as "primary_sources" above
+see_also:
+  - title: Eltoo
+    link: topic eltoo
+
+  - title: Signature adaptors
+    link: topic adaptor signatures
+
+  - title: Multisignatures
+    link: topic multisignature
+
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+## Should be less than 500 characters
+excerpt: >
+  **Statechains** are a proposed offchain system for allowing a user
+  (such as Alice) to delegate the ability to spend a UTXO to another
+  user (Bob), who can then further delegate the spending authority to a
+  third user (Carol), etc.
+
+---
+
+The offchain delegation operations are all performed using [signature
+adaptors][topic adaptor signatures] and the cooperation of a trusted
+third party who can only steal funds if they collude with a delegated
+signer (such as previous delegates Alice or Bob).  A delegated signer
+can always spend the UTXO onchain without needing permission from the
+trusted third party, arguably making statechains less trusted than
+federated [sidechains][topic sidechains].
+
+Because anyone who was ever a delegate can trigger an onchain spend,
+statechains are designed to use the [eltoo][topic eltoo] mechanism to
+ensure an onchain spend by the most recent delegate (Carol) can take
+precedence over spends by previous delegates (Alice and Bob), assuming
+the trusted third party hasn’t colluded with a previous delegate to
```

Comment:
From the text it is not clear to me the third party's base case role in the statechain. I just understand that they cooperate in delegation operations (are they required?) and can only steal funds via collusion w/ any delegate.

Reply:
Rearranged and added some text to hopefully make this clearer.  Let me know if it's not.